File: minecraft/origin/src/main/java/com/github/theword/queqiao/event/minecraft/MinecraftPlayerQuitEvent.java
package com.github.theword.queqiao.event.minecraft;

import com.github.theword.queqiao.tool.event.base.BasePlayerQuitEvent;

public class MinecraftPlayerQuitEvent extends BasePlayerQuitEvent {
    public MinecraftPlayerQuitEvent(MinecraftPlayer player) {
        super("MinecraftPlayerQuitEvent", player);
    }

}

File: minecraft/origin/src/main/kotlin/com/github/theword/queqiao/event/minecraft/MinecraftPlayerQuitEvent.kt
package com.github.theword.queqiao.event.minecraft

import com.github.theword.queqiao.tool.event.base.BasePlayerQuitEvent

data class MinecraftPlayerQuitEvent(
    private val player: MinecraftPlayer
) : BasePlayerQuitEvent("MinecraftPlayerQuitEvent", player)